<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qdUiSettings</class>
 <widget class="QDialog" name="qdUiSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="qtwTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="qcbHorizontalLayout">
         <property name="text">
          <string>&amp;Horizontal layout</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qcbAlwaysOnTop">
         <property name="text">
          <string>Always &amp;on top</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qcbRememberWindowPosition">
         <property name="text">
          <string>Remember window &amp;position</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qcbSystemTrayIcon">
         <property name="text">
          <string>System &amp;tray icon</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QCheckBox" name="qcbShowWordsInTrayBalloon">
           <property name="text">
            <string>Show words in tray &amp;balloon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="qcbMinimizeToTray">
           <property name="text">
            <string>&amp;Minimize to tray</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&amp;Language:</string>
           </property>
           <property name="buddy">
            <cstring>qcbLanguage</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="qcbLanguage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="qcbUpdateCheck">
         <property name="text">
          <string>Check for update</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>&amp;Learning</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QSpinBox" name="qsbWordsFrequency">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string>s</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Words &amp;frequency</string>
           </property>
           <property name="buddy">
            <cstring>qsbWordsFrequency</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QSpinBox" name="qsbWaitForAnswer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string>s</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="qlWaitForAnswer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Wait for &amp;answer</string>
           </property>
           <property name="buddy">
            <cstring>qsbWaitForAnswer</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="qcbNewWordSound">
         <property name="text">
          <string>Play &amp;sound with new word</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QRadioButton" name="qrbSoundSystem">
           <property name="text">
            <string>System</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="qrbSoundCustom">
             <property name="text">
              <string>Custom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="qleSoundCustom"/>
           </item>
           <item>
            <widget class="QPushButton" name="qbpSoundBrowse">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="qcbNewWordFlash">
         <property name="text">
          <string>F&amp;lash with new word</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qcbLearnDisabledWords">
         <property name="text">
          <string>Learn disabled words</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qcbSwitchLearningDirection">
         <property name="text">
          <string>Switch learning &amp;direction</string>
         </property>
         <property name="tristate">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qcbStartLearningOnStartup">
         <property name="text">
          <string>Start learning on s&amp;tartup</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>&amp;Appearance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Main window</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QCheckBox" name="qcbMainWindowToolBar">
            <property name="text">
             <string>Tool bar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="qcbMainWindowLanguageNames">
            <property name="text">
             <string>Language names</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="qcbMainWindowCategoryName">
            <property name="text">
             <string>Category name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="qcbMainWindowRecordControls">
            <property name="text">
             <string>Record controls</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="qcbMainWindowStatusBar">
            <property name="text">
             <string>Status bar</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="ColorComboBox" name="qcbColorFlash">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="qlColorFlash">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Flash &amp;color</string>
              </property>
              <property name="buddy">
               <cstring>qcbColorFlash</cstring>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Vocabulary Manager</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QCheckBox" name="qcbVocabularyManagerCategoriesEnable">
            <property name="text">
             <string>Enable/disable categories</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="qcbVocabularyManagerCategoryPriority">
            <property name="text">
             <string>Change category priority</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>&amp;Hotkeys</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Learning</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&amp;Next</string>
            </property>
            <property name="buddy">
             <cstring>qleHotkeyNext</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="HotkeyLineEdit" name="qleHotkeyNext"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&amp;Answer</string>
            </property>
            <property name="buddy">
             <cstring>qleHotkeyAnswer</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="HotkeyLineEdit" name="qleHotkeyAnswer"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="qpbHotkeyNextClear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="qpbHotkeyAnswerClear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Window</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&amp;Minimize</string>
            </property>
            <property name="buddy">
             <cstring>qleHotkeyMinimize</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="HotkeyLineEdit" name="qleHotkeyMinimize"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&amp;Restore</string>
            </property>
            <property name="buddy">
             <cstring>qleHotkeyRestore</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="HotkeyLineEdit" name="qleHotkeyRestore"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="qpbHotkeyMinimizeClear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="qpbHotkeyRestoreClear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>&amp;Plugins</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Import</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QTreeView" name="qtvPluginsImp">
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <attribute name="headerStretchLastSection">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Export</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QTreeView" name="qtvPluginsExp">
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <attribute name="headerStretchLastSection">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Text to speech</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QTreeView" name="qtvPluginsTTS">
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <attribute name="headerStretchLastSection">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>&amp;Network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QGroupBox" name="qgbUseProxy">
         <property name="title">
          <string>Use proxy</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Host name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="qleProxyHostname"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="qsbProxyPort">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>User name:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="qleProxyUsername"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="qleProxyPassword"/>
          </item>
          <item row="4" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <widget class="QRadioButton" name="qrbProxyTypeHttp">
              <property name="text">
               <string>HTTP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="qrbProxyTypeSocks5">
              <property name="text">
               <string>Socks 5</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="qrbProxyTypeCachingOnlyHttp">
              <property name="text">
               <string>Caching-only HTTP</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="qpbOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qpbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>settings/colorcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>HotkeyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>settingsdialog/hotkeylineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>qtwTabs</tabstop>
  <tabstop>qcbHorizontalLayout</tabstop>
  <tabstop>qcbAlwaysOnTop</tabstop>
  <tabstop>qcbRememberWindowPosition</tabstop>
  <tabstop>qcbSystemTrayIcon</tabstop>
  <tabstop>qcbShowWordsInTrayBalloon</tabstop>
  <tabstop>qcbMinimizeToTray</tabstop>
  <tabstop>qcbLanguage</tabstop>
  <tabstop>qsbWordsFrequency</tabstop>
  <tabstop>qsbWaitForAnswer</tabstop>
  <tabstop>qcbNewWordSound</tabstop>
  <tabstop>qcbNewWordFlash</tabstop>
  <tabstop>qcbSwitchLearningDirection</tabstop>
  <tabstop>qcbStartLearningOnStartup</tabstop>
  <tabstop>qcbColorFlash</tabstop>
  <tabstop>qpbOk</tabstop>
  <tabstop>qpbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>qpbOk</sender>
   <signal>clicked()</signal>
   <receiver>qdUiSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qpbCancel</sender>
   <signal>clicked()</signal>
   <receiver>qdUiSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
